sqlglot/dialects/clickhouse.py

```diff
@@ -105,6 +106,27 @@ def _delta_sql(self: Generator, expression: DATEΤΙΜΕ_DELTA) -> str:
     return _delta_sql
 
 
+def _timestrtotime_sql(self: ClickHouse.Generator, expression: exp.TimeStrToTime):
+    tz = expression.args.get("timezone")
+    datatype = exp.DataType.build(dtype=exp.DataType.Type.TIMESTAMP)
```

Comment:
Nit: style consistency
```suggestion
    datatype = exp.DataType.build(exp.DataType.Type.TIMESTAMP)
```